<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widgetmodel9</class>
 <widget class="QWidget" name="Widgetmodel9">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="XyResultLabel" name="labelTips">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>点击检测设备环境是否正常</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPrint">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>打印标签</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="XyProcessLog" name="tableWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>600</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XyResultLabel</class>
   <extends>QLabel</extends>
   <header>XyResultLabel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XyProcessLog</class>
   <extends>QWidget</extends>
   <header location="global">XyProcessLog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
